config.yaml

```diff
@@ -8,15 +8,17 @@ options:
     type: int
     default: 3306
     description: Port used by MySQL
-  root-password:
+  MYSQL_ROOT_PASSWORD:
     type: string
     default: ''
-    descriptions: Password used by root user
-  idcAdmin-password:
+    descriptions: Set the password for the root user.
+  MYSQL_USER:
```

Comment:
My understanding is that setup like this should be done via a relation.

@jlounder Could you comment?

Reply:
These are parameters you can send to the [image we are using](https://hub.docker.com/r/ubuntu/mysql).
In the specific case of `MYSQL_ROOT_PASSWORD` if you don't set one, the charm will generate one, and will be sent vía relation data to the others units